new.quicktype.ts

```diff
@@ -0,0 +1,208 @@
+/**
```

Comment:
This file is generated by [quicktype](https://quicktype.io/) from the new, hand-authored JSON Schema. The generation is OK _for now_ but if we add something like redirects as described in #91, it has a few inconveniences. We're going to insulate ourselves from quicktype with the next TS module.